docs/elevate/ELevate-NG-testing-guide.md

```diff
@@ -7,30 +7,32 @@ title: "ELevate NG Testing Guide"
 # ELevate NG Testing Guide (Leapp version 0.19.0) 
 
 ::: warning
-Before beginning, we **HIGHLY** recommend that you follow system administration best practices and make sure you have backups and/or snapshots of your system before you proceed. It is recommended to do a trial run in a sandbox to verify that migration worked as expected before you attempt to migrate any production system. Please report any issues encountered to the [AlmaLinux Bug Tracker](https://bugs.almalinux.org) and/or [AlmaLinux Chat Migration Channel](https://chat.almalinux.org/almalinux/channels/migration)
+Before beginning, we **HIGHLY** recommend that you follow system administration best practices and make sure you have backups and/or snapshots of your system before you proceed. It is recommended to do a trial run in a sandbox to verify the upgrade worked as expected before you attempt to upgrade any production system. Please report any issues encountered to the [AlmaLinux Bug Tracker](https://bugs.almalinux.org) and/or [AlmaLinux Chat Migration Channel](https://chat.almalinux.org/almalinux/channels/migration)
 :::
 
 The ELevate NG supports a number of 3rd party repositories:
-* EPEL support is currently available for the migrations to AlmaLinux OS only.
-* Imunify - for migrations to EL 8
-* KernelCare - for migrations to EL 8
+* EPEL support is currently available for upgrades to AlmaLinux OS only.
 * MariaDB - for all supported operating systems 
 * nginx - for all supported operating systems 
 * PostgreSQL - for all supported operating systems 
+* Imunify - for upgrades to EL 8
+* KernelCare - for upgrades to EL 8
 
-Currently, the following migration directions are available:
+Currently, the following upgrade paths are available:
 
 ![image](/images/ELevateNG.svg)
 
-\* - migration from Scientific Linux 7 to AlmaLinux 8 requires a workaround. Please, see more in the [known issues](#known-issues). <br>
-\** - migration to Oracle Linux 9 is available with the [Oracle Leapp utility](https://blogs.oracle.com/linux/post/upgrade-oracle-linux-8-to-oracle-linux-9-using-leapp) and will not be supported by ELevate project.
+\* - upgrading from Scientific Linux 7 to AlmaLinux 8 requires a workaround. Please, see more in the [known issues](#known-issues). <br>
+\** - upgrading to Oracle Linux 9 is available with the [Oracle Leapp utility](https://blogs.oracle.com/linux/post/upgrade-oracle-linux-8-to-oracle-linux-9-using-leapp) and will not be supported by ELevate project.
 
 
-## Migrate CentOS 7 to AlmaLinux 8
+## Upgrade CentOS 7 to AlmaLinux 8
 
 * Update the system to get the latest updates and reboot your machine.
-   ```
-   sudo yum update -y
+   **NOTE:** Since the CentOS 7 repositories are now offline you will need to swap to the CentOS vault, or you can use our CentOS 7 mirror that we've setup for use with ELevate:
+   ```bash
+   curl -o /etc/yum.repos.d/CentOS-Base.repo https://el7.repo.almalinux.org/centos/CentOS-Base.repo
+   yum upgrade -y
```

Comment:
Should we put these updated commands from root as they were `sudo  yum upgrade -y`?

Reply:
Good catch! Fixed that in the latest commit. Mind giving it another look to make sure we're good?